README.md

```diff
@@ -16,7 +16,7 @@ The server will be accesible at `localhost:9000/beeswax/deploy/api/v1`
 A prebuilt docker image is available with:
  
 ```sh
-docker pull jbowen/enrober:v0.0.1
+docker pull jbowen/enrober:v0
```

Comment:
Why change from Major, minor, revision?  I like this format, and seems to be the standard on dockerhub for most images.  We'll probably want to conform to that standard.